src/org.spec.ts

```diff
@@ -184,27 +147,6 @@ describe('org', () => {
   });
 
   describe('updateOrg', () => {
-    test('Valid data should be updated', async () => {
```

Comment:
I deleted this test because there's no data to update. We still check the name is the same in the test below.

Reply:
in this case we don't we need to remove the endpoint at all?